src/pages/publication/files_and_license_tab/FileUploader.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import '@uppy/core/dist/style.css';
+import '@uppy/dashboard/dist/style.css';
+
+import Tus from '@uppy/tus';
+import Uppy from '@uppy/core';
+import { Dashboard } from '@uppy/react';
+import { useTranslation } from 'react-i18next';
+
+const uppy = Uppy({
+  autoProceed: true,
+  restrictions: {
+    allowedFileTypes: ['image/*', 'text/*', 'application/*'],
+  },
+}).use(Tus, {
+  endpoint: 'https://master.tus.io/files/',
+});
+
+interface FileUploaderProps {
+  addFile: (file: any) => void;
+}
+
+const FileUploader: React.FC<FileUploaderProps> = ({ addFile }) => {
+  const { t } = useTranslation('publication');
+
+  uppy.on('upload-success', uploadedFile => {
+    addFile(uploadedFile);
+  });
+
+  return (
+    <Dashboard
+      uppy={uppy}
+      proudlyDisplayPoweredByUppy={false}
+      showSelectedFiles={false}
+      showProgressDetails
+      width={10000}
```

Comment:
hvilken enhet er dette?

Reply:
px :)
https://uppy.io/docs/dashboard/#width-750
svakhet i Uppy imo, hvis man ikke oppgir noe vil ikke default dekke hele bredden som ser vledig rart ut, så satte det bare til "mye"... Vil egentlig ha `width="100%"`, men denne komponenten støtter bare number her (som er litt teit siden dette er en kobinasjon av andre komponenter som faktisk støtter dette, som feks https://uppy.io/docs/drag-drop/#width-39-100-39)

Reply:
om du ikke har gjort det, kan du lage en konstant som forklarer hensikten med tallet og legge med enhet i navnet. (magic number)

Reply:
kan dra ut som egen variabel. Sannsynligvis må vi gjøre litt mer innsats for å få til mer styling her i fremtiden. Dette er jo _egentlig_ en hack

Reply:
eller bare legge til en kammentar bak hvor den brukes? @pcb-unit

Reply:
liker konstanter bedre :-)